src/containerapp/azext_containerapp/_params.py

```diff
@@ -276,6 +276,11 @@ def load_arguments(self, _):
         c.argument('user_assigned', nargs='+', help="Space-separated user identities to be assigned.")
         c.argument('system_assigned', help="Boolean indicating whether to assign system-assigned identity.", action='store_true')
 
+    with self.argument_context('containerapp auth') as c:
+        c.argument('blob_container_uri', help='The URI of the blob storage containing the tokens. Should not be used along with sas_url_secret and sas_url_secret_name.', is_preview=True)
+        c.argument('blob_container_identity', options_list=['--blob-container-identity', '--bci'],
+                   help='Resource ID of a managed identity to authenticate with Azure blob storage, or Empty to use a system-assigned identity.', is_preview=True)
```

Comment:
I wonder if the containerapp doesn't enable system-assigned identity, can it use system-assigned identity for `--blob-container-identity`?

Reply:
RP side will throw exception